code/modules/power/cable.dm

```diff
@@ -38,15 +38,14 @@ By design, d1 is the smallest direction and d2 is the highest
 	var/d2 = 1								// cable direction 2 (see above)
 	plane = ABOVE_TURF_PLANE //Set above turf for mapping preview only, supposed to be ABOVE_PLATING_PLANE, handled in New()
 	layer = WIRE_LAYER
-	var/obj/item/device/powersink/attached	// holding this here for qdel
 	var/_color = "red"
 	color = "red"
 
 	//For rebuilding powernets from scratch
 	var/build_status = 0 //1 means it needs rebuilding during the next tick or on usage
 	var/oldavail = 0
 	var/oldnewavail = 0
-	var/oldload = 0
+	var/list/oldload = new(POWER_PRIORITY_EXCESS)
```

Comment:
list(POWER_PRIORITY_EXCESS)

Not clear why you're starting this at 11 either?

Reply:
No no, I don't want a list that starts with an 11 in it. I want a list with 11 slots, one per priority.
Looking at it I guess I should replace that with a `var/list/oldload[POWER_PRIORITY_EXCESS]` for clarity, right? Same with `var/list/load` in power.dm

Reply:
I see now; yeah makes sense - you might actually go a step further and use a new define TOTAL_PRIORITY_SLOTS and define it as equal to POWER_PRIORITY_EXCESS. This way if someone decides to add another priority level later on, they'll see the slots define in the define file and realize they need to increase it. Otherwise, this is as bad as a magic number because it's not clear why you are using that define (the reason being... that's the largest priority and therefore the last)

Reply:
That bit about `POWER_PRIORITY_EXCESS` being a magic number as used there is fair, yeah. 
Added TOTAL_PRIORITY_SLOTS  and updated cable.dm and power.dm accordingly.